website/docs/cli.md

```diff
@@ -1,8 +1,9 @@
 ---
 id: cli
-title: CLI
 ---
 
+# CLI
```

Comment:
i can rollback this md file change as proper tests are now added

```suggestion
title: CLI
---
```

Reply:
Oh I think it's fine and we should dogfood this a bit too ;)